leo_gz_worlds/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-ament_environment_hooks("${CMAKE_CURRENT_SOURCE_DIR}/hooks/gz_sim_resource_path.dsv.in")
```

Comment:
This file needs the `.in` extension so that ament can replace `@PROJECT_NAME@` in the output dsv file